lib/block-supports/duotone.php

```diff
@@ -5,6 +5,39 @@
  * @package gutenberg
  */
 
+// Register duotone block supports.
```

Comment:
This block looks like it was just moved from below. Is there a reason for that?

Reply:
It's so you don't have to scroll through 1000 lines of deprecated code to see the first line of code that actually does something.